src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -425,6 +429,6 @@ public FormValidation doTestConnection(@QueryParameter URL serverUrl, @QueryPara
 
     @Override
     public String toString() {
-        return MoreObjects.toStringHelper(this).add("name", name).add("serverUrl", serverUrl).toString();
+        return "KubernetesCloud '" + name + "' serverUrl :" + serverUrl;
```

Comment:
let's agree to disagree, I prefer to use helpers and consistent helper methods without code generation